CONTRIBUTING.md

```diff
@@ -31,7 +31,7 @@ This section guides you through submitting a feature request, or enhancement for
 #### Before Submitting An Enhancement Suggestion
 
 * check the issue list for any similar issues
-* take a look at the [RoadMap](ROADMAP.md) or projects to see if the feature is already planned
+* take a look at the [RoadMap](../ROADMAP.md) or projects to see if the feature is already planned
```

Comment:
Why is this going back to the previous directory? Shouldn't it be `./ROADMAP.md`?

Reply:
That is a mistake.  I at one point moved this file to the .github folder, but then started looking through what other projects have done and digging a little and it was strongly suggested that the CONTRIBUTING.md should be in the root folder to be easily found, outside of github.  So I moved it back, and didn't fix all the links.